djaoapp/templates/saas/_invoiceables.html

```diff
@@ -64,4 +65,18 @@
       </tr>
     </tfooter>
   </table>
+  <div id="redeem-code" class="text-right">
```

Comment:
The template for the redeem code should remain in `cart.html` (and not moved here). This is because `_invoiceables.html` is used to show an order details at multiple places.